big_tests/tests/rest_client_SUITE.erl

```diff
@@ -251,6 +278,45 @@ room_is_created_with_given_identifier(Config) ->
         assert_room_info(Alice, RoomInfo)
     end).
 
+config_can_be_changed_by_owner(Config) ->
+    escalus:fresh_story(Config, [{alice, 1}], fun(Alice) ->
+        RoomID = <<"som_othere_id">>,
+        RoomJID = room_jid(RoomID, Config),
+        RoomID = given_new_room({alice, Alice}, RoomJID, <<"old_name">>),
+        RoomInfo = get_room_info({alice, Alice}, RoomID),
+        assert_property_value(<<"name">>,<<"old_name">>,RoomInfo),
+
+        {{<<"200">>, <<"OK">>}, {_}} =
+            given_config_change({alice, Alice}, RoomJID, <<"new_name">>, <<"new_subject">>),
+        NewRoomInfo = get_room_info({alice, Alice}, RoomID),
+        assert_property_value(<<"name">>, <<"new_name">>, NewRoomInfo),
+        assert_property_value(<<"subject">>, <<"new_subject">>, NewRoomInfo)
+    end).
+
+config_cannot_be_changed(Config) ->
```

Comment:
The test should have at least a single word in the name indicating *why* the config cannot be changed.